src/blocks/music.rs

```diff
@@ -25,6 +26,8 @@ pub struct Music {
     dbus_conn: Connection,
     player_avail: bool,
     marquee: bool,
+    marquee_interval: Duration,
```

Comment:
These fields aren't needed, because you can set the values at block 'construction' time in the RotatingTextWidget, as you did. That's also why the compiler is emitting warnings- these fields are never read.